tests/Tests/600__ai_apps/660__openvino/660__openvino.robot

```diff
@@ -26,19 +26,33 @@ Verify OpenVino Is Available In RHODS Dashboard Explore Page
 Verify Openvino Operator Can Be Installed Using OpenShift Console
    [Tags]   Tier2
    ...      ODS-675
+   ...      ODS-652
    ...      ODS-495
    ...      ODS-1236
-   [Documentation]  This Test Case Installed Openvino operator in Openshift cluster
+   [Documentation]  This Test Case Installed Openvino operator in Openshift cluster,
+   ...              Cheks whether RHODS notification shows the Notebook images are running after installing openvino
    ...               and Check and Launch AIKIT notebook image from RHODS dashboard
    Check And Install Operator in Openshift    ${openvino_operator_name}   ${openvino_appname}
    Create Tabname Instance For Installed Operator        ${openvino_operator_name}       Notebook    redhat-ods-applications
    Wait Until Keyword Succeeds    900  1     Check Image Build Status   Complete     openvino-notebook
+   Get Build Status    namespace=redhat-ods-applications    build_search_term=openvino-notebook
+   Click Element    //a[@class="co-resource-item__resource-name"]
+   Wait Until Page Contains    Build details
+   @{link_elements}=  Get WebElements
+   ...    //div[@class='co-m-label-list']
+   @{list} =    Create List
+   FOR  ${link}  IN  @{link_elements}
+         ${txt} =    Get Text    ${link}
+         Append To List    ${list}    ${txt}
+   END
+   Run Keyword And Return Status    Should Contain    ${list}[0]    opendatahub.io/build_type\n=\nnotebook_image
```

Comment:
why are you taking the first element?

Reply:
Labels are stored at index 0

Reply:
what's in the other index then?
The keyword say it returns the label only though

Reply:
there is only one item in the list, I have updated pr